diff -ur pcb-1.6.3/src/dev_rs274x.c pcb-1.6.3p/src/dev_rs274x.c
--- pcb-1.6.3/src/dev_rs274x.c	Wed May 13 06:29:43 1998
+++ pcb-1.6.3p/src/dev_rs274x.c	Mon Jun 14 17:13:55 1999
@@ -48,10 +48,10 @@
 #include <math.h>
 #include <time.h>
 #include <pwd.h>
+#include <stdarg.h>
 #include <sys/types.h>
 #include <errno.h>
 #include <unistd.h>
-#include <varargs.h>
 
 #include "global.h"
 
@@ -218,10 +218,7 @@
 /*----------------------------------------------------------------------------*/
 /* Error Logging Routines                                                     */
 /*----------------------------------------------------------------------------*/
-static void logError(fp, format, va_alist)
-	FILE *fp;
-	char *format;
-	va_dcl
+static void logError(FILE *fp, char *format, ...)
 {
 	va_list args;
 	char    s[1024];
@@ -229,7 +226,7 @@
 		/* FIXME: Is it legitimate to use Message() from within a
 			driver? */
 
-	va_start(args);
+	va_start(args, format);
 	vsprintf(s, format, args);
 	fputs(s, fp);
 /*
diff -ur pcb-1.6.3/src/error.c pcb-1.6.3p/src/error.c
--- pcb-1.6.3/src/error.c	Wed May 13 06:29:45 1998
+++ pcb-1.6.3p/src/error.c	Mon Jun 14 17:15:41 1999
@@ -34,7 +34,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <errno.h>
-#include <varargs.h>
+#include <stdarg.h>
 #include <signal.h>
 #include <unistd.h>
 #include <fcntl.h>
@@ -68,15 +68,13 @@
 /* ---------------------------------------------------------------------------
  * output of message in a dialog window or log window
  */
-void Message(Format, va_alist)
-char *Format;
-va_dcl
+void Message(char *Format, ...)
 {
 	va_list	args;
 	char	s[1024];
 	XEvent	event;
 
-	va_start(args);
+	va_start(args, Format);
 	vsprintf(s, Format, args);
 	va_end(args);
 
@@ -156,9 +154,7 @@
 /* ---------------------------------------------------------------------------
  * output of fatal error message
  */
-void MyFatal(Format, va_alist)
-char *Format;
-va_dcl
+void MyFatal(char *Format, ...)
 {
 	va_list	args;
 
@@ -166,7 +162,7 @@
 	RestoreStderr();
 	EmergencySave();
 
-	va_start(args);
+	va_start(args, Format);
 	fprintf(stderr, "%s (%i): fatal, ", Progname, (int) getpid());
 	vfprintf(stderr, Format, args);
 	fflush(stderr);
diff -ur pcb-1.6.3/src/error.h pcb-1.6.3p/src/error.h
--- pcb-1.6.3/src/error.h	Wed May 13 06:29:45 1998
+++ pcb-1.6.3p/src/error.h	Tue Jun 15 08:49:00 1999
@@ -37,13 +37,13 @@
 #define	STATUS_BREAK	1
 #define	STATUS_ERROR	-1
 
-void	Message();
+void	Message(char*, ...);
 void	OpenErrorMessage(char *);
 void	PopenErrorMessage(char *);
 void	OpendirErrorMessage(char *);
 void	ChdirErrorMessage(char *);
 void	RestoreStderr(void);
-void	MyFatal();
+void	MyFatal(char*, ...);
 void	CatchSignal(int);
 void	X11ErrorHandler(String);
 void	InitErrorLog(void);
